config/pmd.xml

```diff
@@ -162,12 +162,7 @@
       <property name="problemDepth" value="4"/>
     </properties>
   </rule>
-  <rule ref="rulesets/java/design.xml/SwitchDensity">
-    <properties>
-      <!-- till #973 -->
-      <property name="violationSuppressXPath" value="//ClassOrInterfaceDeclaration[@Image='DeclarationOrderCheck']"/>
-    </properties>
-  </rule>
+  <rule ref="rulesets/java/design.xml/SwitchDensity"/>
```

Comment:
@mkordas 
You are right, we have complete design ruleset [here](https://github.com/checkstyle/checkstyle/blob/master/config/pmd.xml#L132) except excludes.